datatree/formatting_html.py

```diff
@@ -0,0 +1,58 @@
+from functools import partial
+from html import escape
+from typing import Any, Mapping
+
+from xarray.core.formatting_html import (
+    _mapping_section,
+    _obj_repr,
+    attr_section,
+    coord_section,
+    datavar_section,
+    dim_section,
+)
+from xarray.core.options import OPTIONS
+
+OPTIONS["display_expand_groups"] = "default"
+
+
+def summarize_children(children: Mapping[str, Any]) -> str:
+    children_li = "".join(
+        f"<ul class='xr-sections'>{node_repr(n, c)}</ul>" for n, c in children.items()
+    )
+
+    return (
+        "<ul class='xr-sections'>"
+        f"<div style='padding-left:2rem;'>{children_li}<br></div>"
+        "</ul>"
```

Comment:
I'm currently padding the sub-groups here to indent them, but I think this is where I would need to add lines instead.